### PR TITLE
Organ defines, get_organ_slot, remove_organ_slot

### PR DESCRIPTION
**Pull request by Ldip999**
## About The Pull Request

Adds organ defines, and for the human_helper procs, adds get_organ_slot and remove_organ_slot

## Why It's Good For The Game

Makes things easier to use, edit, and harder to fuck up, also less lines, also makes porting things from TG easier since they use get_organ_slot and organ defines

## Changelog
:cl:
code: Organ defines, get_organ_slot, remove_organ_slot
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 1]
```diff
@@ -366,3 +366,12 @@
 		return MONKEY_HEIGHT_MEDIUM
 
 	return mob_height
+
+///Gets organ slot by name
+/mob/living/carbon/human/proc/get_organ_slot(var/string)
+	return internal_organs_by_name[string]
+///Removes organ slot by name
```
Comment (QualityVan) [inline code/modules/mob/living/carbon/human/human_helpers.dm:373]: Spacing

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 2]
```diff
@@ -403,12 +403,12 @@ Contains most of the procs that are called when a mob is attacked by something
...
 	if(!do_after(user, 2 SECONDS, NONE, src))
 		return
-	if(!internal_organs_by_name["heart"])
+	if(!get_organ_slot(ORGAN_SLOT_HEART))
 		to_chat(user, span_notice("The heart is no longer here!"))
 		return
 	log_combat(user, src, "ripped [src]'s heart", I)
 	visible_message(span_notice("[user] ripped off [src]'s heart!"), span_notice("You ripped off [src]'s heart!"))
-	internal_organs_by_name -= "heart"
+	internal_organs_by_name -= ORGAN_SLOT_HEART
```
Comment (QualityVan) [inline code/modules/mob/living/carbon/human/human_defense.dm:411]: Shouldn't this be using remove?

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 3]
```diff
@@ -366,3 +366,12 @@
 		return MONKEY_HEIGHT_MEDIUM
 
 	return mob_height
+
+///Gets organ slot by name
+/mob/living/carbon/human/proc/get_organ_slot(var/string)
```
Comment (QualityVan) [inline code/modules/mob/living/carbon/human/human_helpers.dm:371]: For both this and remove it's not actually getting or removing the slot, right? It's returning the organ itself.

**Review by QualityVan — CHANGES_REQUESTED** [review]
This is a lot of replacing list access with proc calls which is some extra processing, but most of this is infrequent enough that it shouldn't really matter.
Check linters.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**